controlplane/kubeadm/internal/workload_cluster_coredns_test.go

```diff
@@ -284,44 +283,64 @@ kind: ClusterConfiguration
 		},
 	}
 
+	// We are using testEnv as a workload cluster, and given that each test case assumes well known objects with specific
+	// Namespace/Name (e.g. The CoderDNS ConfigMap & Deployment, the kubeadm ConfigMap), it is not possible to run the use cases in parallel.
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			g := NewWithT(t)
-			fakeClient := fake.NewFakeClientWithScheme(scheme.Scheme, tt.objs...)
-			w := &Workload{
-				Client:          fakeClient,
-				CoreDNSMigrator: tt.migrator,
-			}
-			err := w.UpdateCoreDNS(ctx, tt.kcp)
-			if tt.expectErr {
-				g.Expect(err).To(HaveOccurred())
-				return
-			}
-			g.Expect(err).ToNot(HaveOccurred())
+		g := NewWithT(t)
```

Comment:
I wonder if it would be helpful to print out `tt.name` with `t.Log` or something.

Reply:
done